CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # App Center SDK for Android Change Log
 
-## Version 2.3.1 (Under development)
+## Version 2.4.0 (Under development)
+
+### App Center Auth
+
+* **[Feature]** Add support for Azure Active Directory (AAD) type authentication. Users with AAD apps on Azure can authenticate using one of three audiences viz. AzureADMyOrg (Single Tenant), AzureADMultipleOrgs (Multi Tenant) and AzureADandPersonalMicrosoftAccount (Multi Tenant and Personal Microsoft accounts).
```

Comment:
Still needs to be reviewed by the feature's PM.

Reply:
Nice writing! It’s not immediately clear to me (and I’d imagine the user) what the terms viz, AzureADMyOrg, AzureADMultipleOrgs, AzureADandPersonalMicrosoftAccount means. What I’d recommend is to explain that explicitly in the language, and add a link for the users to learn more :)

Question: 
- is there a link that explains AAD app types single tenant, multi-tenant and personal MSA accounts? I could only find this, but that doesn’t capture personal MSA accounts https://docs.microsoft.com/en-us/azure/active-directory/develop/app-types#single-tenant-and-multi-tenant-apps. Would be good to include a link.
- what are the terms “azureADMyOrg” etc? Would be good if there are links to explain these terms as well. The general guidance of a good description is that it should be instantly clear to the layman user, with links to learn more if the terms are more complex.

Here’s a proposal, and if you have the links please link it below:
```suggestion
* **[Feature]** Add authentication support for Azure Active Directory (AAD) applications. Users can now connect an [AAD tenant and AAD single tenant (AzureADMyOrg), multi-tenant (AzureADMultipleOrgs) and multi-tenant and personal Microsoft account (AzureADandPersonalMicrosoftAccount)](https://www.xxx.com) applications.
```